config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
-  devise_for :users
+  mount_devise_token_auth_for 'User', at: 'auth'
```

Comment:
This is also from last PR

Reply:
Are you forgetting a rebase or a pull/push ?

Reply:
I do branches from the last branch that depends. I don't do pull/push and I do rebase just when you accept my PR.